src/echo.ts

```diff
@@ -95,7 +102,7 @@ export default class Echo {
      * Get a private encrypted channel instance by name.
      */
     encryptedPrivate(channel: string): Channel {
-        return this.connector.encryptedPrivateChannel(channel);
+        return (this.connector as NullConnector | PusherConnector).encryptedPrivateChannel(channel);
```

Comment:
Not sure why this is here socket.io connector is not implementing it. (The user can always just access `echo.connector`)